src/app/loadout-builder/process-worker/process-utils.ts

```diff
@@ -220,18 +244,20 @@ export function canTakeSlotIndependantMods(
           const notLegacySocketOrLegacyMod = !item.hasLegacyModSocket || !otherP[i];
 
           raidsFit &&= Boolean(
-            noRaidMod ||
-              (raidEnergyIsValid &&
-                notLegacySocketOrLegacyMod &&
-                raidTag &&
-                item.compatibleModSeasons?.includes(raidTag))
+            raidEnergyIsValid &&
+              notLegacySocketOrLegacyMod &&
+              raidTag &&
+              item.compatibleModSeasons?.includes(raidTag)
           );
-        }
 
-        if (!raidsFit) {
-          break;
```

Comment:
This was the issue I believe, it was breaking the "mods" loop and not checking other permutations.